test/test_autograd.py

```diff
@@ -9524,30 +9524,6 @@ def test_foward_mode_AD(self, device):
             else:
                 torch._test_autograd_multiple_dispatch(dual_input)
 
-    def test_view_copy(self, device):
-        # tests that view_copy derivative formulas are also generated per dispatch key
-        # from their respective view ops in derivatives.yaml
-        t = torch.randn(2, 2, device=device, requires_grad=True)
-        t_ref = t.clone().detach().requires_grad_()
-        # _test_autograd_multiple_dispatch_view does a .view(-1) on the input
-        t_view = torch._test_autograd_multiple_dispatch_view(t_ref)
-        t_view_copy = torch._test_autograd_multiple_dispatch_view_copy(t)
```

Comment:
This test is not covered by OpInfo. It seems to be special test for this op. We should probably keep it.

cc: @mikaylagawarecki (original author) @bdhirsh

Reply:
This test checks that per-dispatch key view_copy autograd formulas in derivatives.yaml are correctly codegened from their copy variants (e.g. if the view variant derivative formula is registered to `AutogradCUDA` then the view copy formula is correctly registered). I don't think it is covered by opinfo so I agree that we should keep it